maven build fails when the path to source contains a blank

### Jenkins and plugins versions report

<details>
  <summary>Environment</summary>

  <!-- Paste your environment details below -->
  ```text
version 3.38 of maven-hpi-plugin

  ```

</details>


### What Operating System are you using (both controller, and any agents involved in the problem)?

Windows

### Reproduction steps

1. put the sources of your plugin in a folder that contains a blank, e.g. c:\data\dev\jenkins\my plugins\myplugin
2. run mvn clean install

### Expected Results

The build succeeds

### Actual Results

the build fails with 
`Error: Could not find or load main class plugins\pipeline-groovy-lib-plugin\target\patch-modules\org-netbeans-insane-hook.jar`

`[ERROR] Command was cmd.exe /X /C ""...  --add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.io=ALL-UNNAMED --add-opens java.base/java.util=ALL-UNNAMED --add-opens java.desktop/com.sun.beans.introspect=ALL-UNNAMED --patch-module=java.base=C:\data\dev\jenkins\my plugins\pipeline-groovy-lib-plugin\target\patch-modules\org-netbeans-insane-hook.jar --add-exports=java.base/org.netbeans.insane.hook=ALL-UNNAMED ..." #`

### Anything else?

_No response_

--- a/src/main/java/org/jenkinsci/maven/plugins/hpi/TestRuntimeMojo.java
+++ b/src/main/java/org/jenkinsci/maven/plugins/hpi/TestRuntimeMojo.java
@@ -131,7 +131,7 @@ public class TestRuntimeMojo extends AbstractJenkinsMojo {
 
         Path insaneHook = getInsaneHook(wrap(jth), patchModuleDir.toPath());
 
-        String argLine = String.format("--patch-module=java.base=%s --add-exports=java.base/org.netbeans.insane.hook=ALL-UNNAMED", insaneHook);
+        String argLine = String.format("--patch-module='java.base=%s' --add-exports=java.base/org.netbeans.insane.hook=ALL-UNNAMED", insaneHook);
         getLog().info("Setting jenkins.insaneHook to " + argLine);
         project.getProperties().setProperty("jenkins.insaneHook", argLine);
     }

